BREAKING CHANGE(web-twig): Set isExpandedOnMobile in ModalDialog to false by default #DS-1201 (#1387)
lmc-eu/spirit-design-system: packages/web-react/src/components/Modal/README.md

  - [Opening the Modal](#opening-the-modal)
  - [Scrolling Long Content](#scrolling-long-content)
    - [Scrolling with ScrollView](#scrolling-with-scrollview)
    - [Disable Scrolling Inside ModalDialog](#disable-scrolling-inside-modaldialog)
  - [Stacking Modals](#stacking-modals)

>   - [Full Example](#full-example)

Reviewer: Thanks for completing the TOC!

### API

| Name                          | Type                  | Default   | Required | Description                                                                                                                              |
| ----------------------------- | --------------------- | --------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
| `children`                    | `ReactNode`           | —         | ✕        | Children node                                                                                                                            |
| `elementType`                 | [`article` \| `form`] | `article` | ✕        | ModalDialog element type                                                                                                                 |
| `isDockedOnMobile`            | `bool`                | `false`   | ✕        | [REQUIRES FEATURE FLAG](#feature-flag-uniform-appearance-on-all-breakpoints): Dock the ModalDialog to the bottom of the screen on mobile |

> | `isExpandedOnMobile`          | `bool`                | `false`   | ✕        | ModalDialog shrinks to fit the height of its content.                                                                                    |

Reviewer: Please fix the description, now it says the opposite 🙂.

```suggestion
| `isExpandedOnMobile`          | `bool`                | `false`   | ✕        | If true, ModalDialog expands to fit the viewport                                                                                    |
```

(With no full stops anywhere, I believe?)

Reviewer: … on mobile, of course.
